articles/policies/rate-limits.md

```diff
@@ -319,3 +319,17 @@ For database connections, Auth0 limits certain types of repeat login attempts de
 ## Limits on SMS Messages for MFA
 
 There's a limit of 10 SMS messages/hour per user for multi-factor authentication. For more information, see [Configuring Twilio for SMS](/multifactor-authentication/twilio-configuration).
+
+## Limits on Sign In with Apple 
```

Comment:
can this be renamed to Limits on "Native Social Login" or "Native Sign In with Apple"?
The current tifle might lead to some confusion since it does not apply to sign in with apple when using the web flow